core/blockchain.go

```diff
@@ -45,7 +45,7 @@ import (
 	"github.com/ava-labs/coreth/core/state/snapshot"
 	"github.com/ava-labs/coreth/internal/version"
 	"github.com/ava-labs/coreth/params"
-	customrawdb "github.com/ava-labs/coreth/plugin/evm/rawdb"
+	"github.com/ava-labs/coreth/plugin/evm/customrawdb"
 	customtypes "github.com/ava-labs/coreth/plugin/evm/types"
```

Comment:
should we do the same renaming for types as well?

Reply:
Sure, this makes sense, on it.

Reply:
Done in https://github.com/ava-labs/coreth/pull/890